### ISSUE TITLE
Setting custom folder icons doesn't work

### ISSUE THREAD

**Issue by mxmlnkn** [linuxmint/nemo#1154]
After updating to nemo 2.8.8 all my custom folder icons I set disappeared and I can't set them anew with nemo by doing:
RMB on folder -> Properties -> click on folder icon -> choose .ico -> press open
Nothing changes after this.

Note that nautilus has a similar problem, but even worse, after clicking on the folder icon the file-chooser window opens, but it ignores every click I do. Only Esc-works to close it again.
In both cases there don't seem to be any relevant error messages, only this one https://github.com/linuxmint/nemo/issues/1153

When trying to set the icons in the shell like I did before it suddenly doesn't work anymore either:

    gvfs-set-attribute . metadata::custom-icon 1.ico

yields

    Error setting attribute: Setting attribute metadata::custom-icon not supported

I'm using gvfs 1.28.1

**Comment by Athest** [comment]
I am having the same problem with Nautilus. This started after updating gnome to 3.20. 
Previously set icons are still there, but i am not able to set new icons. ESC key quits, Enter key does not do anything.

I am running Arch Linux, fully updated.

**Comment by mxmlnkn** [comment]
I forgot to update this issue, but for me the reason for the `custom-icon not supported` error was, because I tried to compile and install glib from source, see also here: https://bugzilla.gnome.org/show_bug.cgi?id=765586 a simple `make uninstall` solved that problem.

@athest: But I tried the problem with nautilus again, and the bug with not being able to click or press any key in the icon-chooser menu still is there. I think it would be better to open a bug to the [nautilus bugtracker](https://bugzilla.gnome.org/page.cgi?id=browse.html&product=nautilus) and link it here and close this issue. Sorry for the inconvenience, I guess you find this issue using Google.
